Assets/MRTK/Providers/LeapMotion/LeapMotionArticulatedHand.cs

```diff
@@ -8,11 +8,13 @@
 using System;
 using Unity.Profiling;
 
+#if !WINDOWS_UWP || UNITY_EDITOR
```

Comment:
Is this needed? I believe this assembly is only marked for the editor and Windows standalone, so I don't think it even attempts to build for UWP during compilation. The change in LeapMotionConfigurationChecker.cs might be all that's needed?

Reply:
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/9e60cb3f7850f61bb91a0dc2b5016e42ca54930f/Assets/MRTK/Providers/LeapMotion/Microsoft.MixedReality.Toolkit.Providers.LeapMotion.asmdef#L6-L10

Reply:
Didn't notice the LeapMotionProvider is built to a separate assembly. Removed the guards.